dired-rifle.el

```diff
@@ -42,11 +42,20 @@
           (const :tag "Default" nil)
           (string :tag "Custom")))
 
+(defcustom dired-rifle-use-marked-files nil
+  "If non-nil, dired-rifle will work on marked files instead of the
+file under point. But 'file under point' will be used as fallback, if
+there are no marked files"
+  :type '(choice
+          (const :tag "single-file" nil)
+          (const :tag "marked-files" t))
+  )
+
 (defun rifle-args (&rest args)
   "Return all the common args for rifle along with ARGS as a list."
   (append (when rifle-config
             (list "-c" (expand-file-name rifle-config)))
-          args))
+          (flatten-tree args)))
```

Comment:
What's the purpose of `flatten-tree` here? In which scenario is `args` not a flat list?

Reply:
I do not remember. Seems like I was thinking it is necessary, but obviously it is not.